website/docs/reference/dbt-jinja-functions/ref.md

```diff
@@ -119,3 +124,5 @@ Another example is when a reference appears within an [`is_incremental()`](/docs
 select * from {{ source('raw', 'orders') }}
 {% endif %}
 ```
+
+Note that to ensure dbt understands the dependency, use a SQL comment instead of a Jinja comment. Jinja comments (`{# ... #}`) _don't_ work and are ignored by dbt's parser, meaning `ref` is never processed and resolved. SQL comments, however, (`--` or `/* ... */`) _do_ work because dbt still evaluates Jinja inside SQL comments.
```

Comment:
```suggestion
:::tip
To ensure dbt understands the dependency, use a SQL comment instead of a Jinja comment. Jinja comments (`{# ... #}`) _don't_ work and are ignored by dbt's parser, meaning `ref` is never processed and resolved. SQL comments, however, (`--` or `/* ... */`) _do_ work because dbt still evaluates Jinja inside SQL comments.
:::
```